### PR TITLE
fix a few function names on comments

### PR DESCRIPTION
**Pull request by cuishuang**

### CODE REVIEW

File: pkg/scrape/config/config_http.go [hunk 1]
```diff
@@ -35,7 +35,6 @@ import (
 	"sync"
 	"time"
 
-	"github.com/mwitkow/go-conntrack"
```
Comment (kolesnikovae) [inline pkg/scrape/config/config_http.go:38]: Must've been removed unintentionally. This import is required

File: pkg/util/process/process_unix.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ func Exists(pid int) bool {
 	return s != "Z"
 }
 
-// SendProcess send signal s to the process p.
+// SendSignal send signal s to the process p.
```
Comment (kolesnikovae) [inline pkg/util/process/process_unix.go:27]:
```suggestion
// SendSignal sends signal s to the process p.
```

**Review by kolesnikovae — CHANGES_REQUESTED** [review]
Thank you for the contribution @cuishuang! There is one issue with a removed import – could you check it please?

**Review by Rperry2174 — APPROVED** [review]
LGTM! Thanks @cuishuang !

**Pull request merged**